subgraphs/ellipsis-finance/subgraph.template.yaml

```diff
@@ -1,400 +0,0 @@
-{{#graftEnabled}}
```

Comment:
Why remove this template file?

Reply:
This was a duplicate in the subgraphs root folder. There is a copy under protocols/templates that you can see.